docs/source/cloud_integration.md

```diff
@@ -1,11 +1,11 @@
-ETL
-===
+Cloud Integration
```

Comment:
Renaming the file will break the docs without also updating the TOC in docs/source/index.rst

Reply:
@benfred - This doc should now appear under Resources. I'll remove it completely from docs/source/index.rst.